src/Felixkiss/UniqueWithValidator/ValidatorExtension.php

```diff
@@ -34,7 +34,7 @@ public function validateUniqueWith($attribute, $value, $parameters)
         $parameters = array_map('trim', $parameters);
 
         // first item equals table name
-        $table = array_shift($parameters);
+        list($connection, $table) = $this->parseTable(array_shift($parameters));
```

Comment:
Sorry for the delay. I will try to find time to review this in the near future. You can use your fork in the meantime. Cheers.